src/platform/plugins/shared/discover/public/application/main/state_management/redux/constants.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import type { TabItem } from '@kbn/unified-tabs';
+import type { TabState } from './types';
+
+export const DEFAULT_TAB_STATE: Omit<TabState, keyof TabItem> = {
```

Comment:
I can't even remember exactly why I moved this at this point tbh... Pretty sure I was running into circular dependency issues with these changes.